auth/standard_role_manager.hh

```diff
@@ -17,6 +17,7 @@
 #include <seastar/core/future.hh>
 #include <seastar/core/sstring.hh>
 
+#include "cql3/statements/describe_statement.hh"
```

Comment:
It's nasty to pull in CQL into the auth interface like this.

We can't completely divorce it from CQL (it is producing CQL), but we can isolate it from everything else. Please isolate the description thing into its own header, put it in cql3/ (not cql3/statements, it's not a statement in itself), and make sure it has minimal dependencies on the rest of CQL.

Reply:
And beyond that, use a forward declaration in auth/*.hh. Only include it in .cc, if possible.